app/assets/javascripts/initializers/initializeSponsorshipVisibility.js

```diff
@@ -1,6 +1,4 @@
-'use strict';
-
-const sponsorClickHandler = event => {
+function sponsorClickHandler(event) {
```

Comment:
So using a function instead so it just gets loaded again instead of getting the `VM104:1 Uncaught SyntaxError: Identifier 'sponsorClickHandler' has already been declared`.

This is a nice bandaid solution, but we should investigate why it's being loaded twice.